crates/modelardb_embedded/bindings/python/modelardb/operations.py

```diff
@@ -56,32 +59,45 @@ def __find_library(build: str) -> str:
 
             :raises RuntimeError: If it is not executed on Linux, macOS, or Windows.
             """
-            script_folder = os.path.dirname(os.path.abspath(__file__))
+            # Attempt to load the library installed as part of the Python package.
+            library_folder = pathlib.Path(__file__).parent.parent.resolve()
+
+            if sys.platform == "win32":
+                # SHLIB_SUFFIX is not set and .pyd is used by Rust.
+                library_name = "modelardb_embedded.pyd"
+            else:
+                library_name = "modelardb_embedded" + sysconfig.get_config_var(
+                    "SHLIB_SUFFIX"
+                )
+
+            library_path = library_folder / library_name
+            if library_path.exists():
+                return library_path
+
+            # Attempt to load the library compiled in the development repository.
+            repository_root = pathlib.Path(
+                __file__
+            ).parent.parent.parent.parent.parent.parent.resolve()
+            library_folder = repository_root / "target" / build
 
-            # Select the library to use based on the operating system.
             match platform.system():
                 case "Linux":
```

Comment:
Since the handling for `Linux` and `FreeBSD` are the same, maybe use `case "Linux" | "FreeDSB"`.